README.md

```diff
@@ -20,6 +20,54 @@ Fork this repository and add your tables here:
 
 (your tables go here - you can use TableGenerator to create them. Include the data from the sample above in your tables)
 
+### Blog Table
+| Id | Slug                          | Title                                | Body           | Views |
+|----|-------------------------------|--------------------------------------|----------------|-------|
+| 1  | serilog-structured-logging    | Structured Logging with Serilog      | Long Text Here | 123   |
+| 2  | persisting-smart-enum         | Persisting a Smart Enum with EF Core | Long Text Here | 234   |
+| 3  | localization-aspnet-core-apis | Localization in ASP.NET Core APIs    | Long Text Here | 345   |
+
+### Author Table
+| Id | AuthorName    | AuthorEmail                 | AuthorBio      | Blog_Id |
+|----|---------------|-----------------------------|----------------|---------|
+| 1  | Eric Fleming  | ericfleming@nimblepros.com  | Long Text Here | 1       |
+| 2  | Kyle McMaster | kylemcmaster@nimblepros.com | Long Text Here | 2       |
+| 3  | Steve Smith   | steve@nimblepros.com        | Long Text Here | 3       |
```

Comment:
Not sure about this foreign key direction with Blog_Id here. What this is saying is that you can't have an author without a blog, an author can have exactly one blog, and a blog can exist with 0, 1, or many authors. If that's what you want, fine, but it seems odd to me.